sqlglot/dialects/mysql.py

```diff
@@ -32,6 +32,10 @@
 from sqlglot.tokens import TokenType
 
 
+class ZeroFill(exp.Expression):
+    pass
+
+
```

Comment:
We shouldn't define new expression types outside of `expressions.py`. Please move this to said module and define it as `ZeroFillColumnConstraint(ColumnConstraint): arg_types = {}`, instead.